test/e2e/tests/tokens/nft/erc1155-interaction.spec.js

```diff
@@ -38,8 +48,8 @@ describe('ERC1155 NFTs testdapp interaction', function () {
         await driver.clickElement('#batchMintButton');
 
         // Notification
-        const windowHandles = await driver.waitUntilXWindowHandles(3);
-        const [extension] = windowHandles;
+        let windowHandles = await driver.waitUntilXWindowHandles(3);
+        let [extension] = windowHandles;
         await driver.switchToWindowWithTitle(
           WINDOW_TITLES.Dialog,
           windowHandles,
```

Comment:
```suggestion
        await driver.switchToWindowWithTitle(WINDOW_TITLES.Dialog);
```
and we should remove the ); on line 56 (I cannot edit it from the github ui unfortunately)